src/components/panel/panel.tsx

```diff
@@ -150,6 +152,10 @@ export class Panel implements ConditionalSlotComponent, InteractiveComponent {
 
   panelScrollEl: HTMLElement;
 
+  resizeObserver = !Build.isTesting
```

Comment:
Was causing an error in the `slot` commonTest if this was not there.

Reply:
Correct. slot helper error will appear. I'll check it in so you can see. I'm seeing if I can figure out why.